xchange-bitfinex/src/main/java/org/knowm/xchange/bitfinex/service/BitfinexAccountServiceRaw.java

```diff
@@ -39,20 +42,26 @@
    *
    * @param exchange
    */
-  public BitfinexAccountServiceRaw(Exchange exchange) {
+  public BitfinexAccountServiceRaw(
+      BitfinexExchange exchange, ResilienceRegistries resilienceRegistries) {
 
-    super(exchange);
+    super(exchange, resilienceRegistries);
   }
 
   public BitfinexTradingFeeResponse[] getBitfinexDynamicTradingFees() throws IOException {
```

Comment:
for these types of methods can you just return the body inline instead of setting to a variable `BitfinexTradingFeeResponse[] response` etc